# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1420331146 -3600
#      Sun Jan 04 01:25:46 2015 +0100
# Branch gui-release
# Node ID c2d01ed114bae59ef8679f33b59f138b0e78c48a
# Parent  17e6c770e6acac16b931e98809e6e8222f842f74
fix connecting selection signal of editor to enabling search in selection

* find-dialog.h: remove #ifdef for moc-file generation

diff --git a/libgui/src/m-editor/find-dialog.h b/libgui/src/m-editor/find-dialog.h
--- a/libgui/src/m-editor/find-dialog.h
+++ b/libgui/src/m-editor/find-dialog.h
@@ -74,20 +74,19 @@ class QPushButton;
 class find_dialog : public QDialog
 {
   Q_OBJECT
 public:
   find_dialog (QsciScintilla* edit_area, QWidget *parent = 0);
   void init_search_text ();
 
 private slots:
-#ifdef HAVE_QSCI_FINDSELECTION
+
   void handle_sel_search_changed (int);
-  void handle_selection_changed (bool has_selected);
-#endif
+  void handle_selection_changed (bool);
   void handle_backward_search_changed (int);
   void handle_search_text_changed (QString new_search_text);
 
   void find (bool forward = true);
   void find_next ();
   void find_prev ();
   void replace ();
   void replace_all ();
